sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -466,7 +481,7 @@ void setSQLiteOpenHelper(@NonNull SQLiteOpenHelper helper) {
         mSQLiteOpenHelper = helper;
     }
 
-    private void createTable(SQLiteDatabase db, String table, ContentValues schema) {
+    private void createTable(SQLiteDatabase db, String table, ContentValues schema, String[] uniqueColumnsConstraint) {
```

Comment:
Won't the unique columns always be the same?  I wouldn't think this needed to be a parameter

Reply:
The DatabaseManager from core module handles database interactions with sqlite3 while stay away from all the storage business logic in Storage module. I think it makes sense to extend this `createTable` API with a new parameter so that it's also flexible for usage in other modules in the future.